files/image_config/rsyslog/rsyslog-config.sh

```diff
@@ -25,8 +25,18 @@ fi
 
 hostname=$(hostname)
 
+syslog_with_osversion=$(sonic-db-cli CONFIG_DB hget "DEVICE_METADATA|localhost" "syslog_with_osversion")
```

Comment:
>syslog_with_osversion [](http://example.com/codeflow?start=81&length=21)

Need a daemon to monitor this config change, and take actions like restarting rsyslog service.

Reply:
Will add monitor and update code to hostcfgd

Reply:
Fixed: https://github.com/sonic-net/sonic-host-services/pull/245